cmd/controller/main.go

```diff
@@ -167,7 +170,7 @@ func main() {
 
 	// Start all of the controllers.
 	logger.Info("Starting controllers.")
-	go kncontroller.StartAll(stopCh, controllers...)
+	go kncontroller.StartAll(stopCh, controllers[:]...)
```

Comment:
Just FYI this will conflict with #1118 which is also yours :)

Reply:
Yep, the cost of separating the two different issues :)